packages/e2e/cypress/e2e/login.cy.ts

```diff
@@ -10,28 +10,28 @@ describe('Login', () => {
     it('Should login successfully', () => {
         cy.logout();
         cy.visit('/login');
-        cy.findByLabelText('Email address *').type(
+        cy.findByPlaceholderText('Your email address').type(
             SEED_ORG_1_ADMIN_EMAIL.email,
         );
-        cy.findByLabelText('Password *').type(
+        cy.findByPlaceholderText('Your password').type(
             SEED_ORG_1_ADMIN_PASSWORD.password,
         );
-        cy.get('[data-cy="login-button"]').click();
+        cy.get('form').findByText('Sign in').click();
         cy.url().should('include', '/home');
     });
     it('Should display error message when credentials are invalid', () => {
         cy.logout();
         cy.visit('/login');
-        cy.findByLabelText('Email address *').type('test-email');
-        cy.findByLabelText('Password *').type('test-password');
-        cy.get('[data-cy="login-button"]').click();
+        cy.findByPlaceholderText('Your email address').type('test-email');
+        cy.findByPlaceholderText('Your password').type('test-password');
+        cy.get('form').findByText('Sign in').click();
```

Comment:
just in case find button here instead of a text

Reply:
because we already have a Sign in as a header text and it was little bit confusing

![CleanShot 2023-03-22 at 20 53 55@2x](https://user-images.githubusercontent.com/962095/226979719-03757f4d-10a9-441a-86fe-467292c76721.png)